extensions/amp-story/0.1/media-tasks.js

```diff
@@ -0,0 +1,459 @@
+/**
```

Comment:
I think eventually we'll want to have all these in a ``media-tasks`` folder.

Reply:
in separate files*

Reply:
+1